landing/src/app.css

```
@@ -0,0 +1,375 @@
+.footer-link {
+  position: relative;
+}
+
+.footer-link::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 1px;
+  background: currentColor;
```

Stylelint (17.2.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)